Big-Spender/readme.md

```diff
@@ -124,7 +131,13 @@ CREATE YOUR QUERY HERE
 **You:** Sure thing. To confirm, the date is August 19, 2021, the transaction number is 38104091, the supplier invoice number is 3780119655, the supplier is 'Dell', the expense type is 'Hardware' and the expense area is 'IT'. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERIES HERE
+INSERT INTO spends (description, amount, date, transaction_no, supplier_inv_no) values ('Computer Hardware Dell', 32000, '2021-08-19', 38104091,3780119655);
+INSERT INTO suppliers (supplier) values ('Dell');
+INSERT INTO expense_areas (expense_area) values ('IT');
+UPDATE spends
```

Comment:
A different approach is to create the spends record at the end when you have all the required IDs you want to insert.